k8s.gcr.io/manifests/k8s-staging-sig-storage/promoter-manifest.yaml

```diff
@@ -0,0 +1,10 @@
+# google group for gcr.io/k8s-staging-sig-storage is k8s-infra-staging-sig-storage@kubernetes.io
+registries:
+- name: gcr.io/k8s-staging-sig-storage
+  src: true
+- name: us.gcr.io/k8s-artifacts-prod/storage
```

Comment:
Was it intentional or an oversight that the final destination is still "storage" instead of "sig-storage"?

/cc @msau42

Reply:
I think it was a mistake

Reply:
It was a miss from my side @pohly. Raised this PR to fix it. https://github.com/kubernetes/k8s.io/pull/971